finesse/gui/hardware_set/finesse_dp9800.yaml

```diff
@@ -3,7 +3,7 @@ devices:
   stepper_motor:
     class_name: stepper_motor.st10_controller.ST10Controller
     params:
-      port: "0403:6011 FT1NMSVR (4)"
```

Comment:
So, where does this `(4)` came from?

Reply:
The extra number at the end is to distinguish ports which have the same vendor ID, product ID and serial number. It sounds odd, but you see this with USB serial adapters that have multiple ports (Jon is using one). The ports will always be in the same order for a given device.